lib/js/jsconsole.nim

```diff
@@ -25,45 +25,69 @@ proc debugImpl(console: Console) {.importcpp: "debug", varargs.}
   makeConsoleCall(console, bindSym "debugImpl", args)
 
 macro info*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/info
   makeConsoleCall(console, bindSym "infoImpl", args)
 
 macro error*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/error
   makeConsoleCall(console, bindSym "errorImpl", args)
 
+macro exception*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/error
+  makeConsoleCall(console, bindSym "errorImpl", args)
+
+macro trace*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
+  ## https://developer.mozilla.org/docs/Web/API/Console/trace
+  makeConsoleCall(console, bindSym "traceImpl", args)
 
 macro warn*(console: Console, args: varargs[RootRef, convertToConsoleLoggable]): untyped =
-  ## https://developer.mozilla.org/en-US/docs/Web/API/Console/warn
+  ## https://developer.mozilla.org/docs/Web/API/Console/warn
   makeConsoleCall(console, bindSym "warnImpl", args)
 
-proc clear*(console: Console) {.importcpp: "clear".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/clear
+proc clear*(console: Console) {.importcpp: "clear".}
+  ## https://developer.mozilla.org/docs/Web/API/Console/clear
+
+proc count*(console: Console, label = "".cstring) {.importcpp: "count".}
+  ## https://developer.mozilla.org/docs/Web/API/Console/count
 
-proc count*(console: Console, label = "".cstring) {.importcpp: "count".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/count
+proc countReset*(console: Console, label = "".cstring) {.importcpp: "countReset".}
+  ## https://developer.mozilla.org/docs/Web/API/Console/countReset
 
-proc countReset*(console: Console, label = "".cstring) {.importcpp: "countReset".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/countReset
+proc group*(console: Console, label = "".cstring) {.importcpp: "group".}
+  ## https://developer.mozilla.org/docs/Web/API/Console/group
 
-proc group*(console: Console, label = "".cstring) {.importcpp: "group".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/group
+proc groupCollapsed*(console: Console, label = "".cstring) {.importcpp: "groupCollapsed".}
+  ## https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed
 
-proc groupCollapsed*(console: Console, label = "".cstring) {.importcpp: "groupCollapsed".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/groupCollapsed
+proc groupEnd*(console: Console) {.importcpp: "groupEnd".}
+  ## https://developer.mozilla.org/docs/Web/API/Console/groupEnd
 
-proc groupEnd*(console: Console) {.importcpp: "groupEnd".} ## https://developer.mozilla.org/en-US/docs/Web/API/Console/groupEnd
+proc time*(console: Console, label = "".cstring) {.importcpp: "time".}
```

Comment:
these seem purely for documentation purposes, looks like for js target, `importcpp` semantics makes calling these work regardless whether they're declared in nim. However I  still think these are good to have.

But everywhere in this file please remove redundant importcpp name when it matches the proc name, eg:
```nim
proc time*(console: Console, label = "".cstring) {.importcpp: "time".}
=>
proc time*(console: Console, label = "".cstring) {.importcpp.}
```

you could also wrap these inside a push/pop importcpp (simpler is better!)